app/workers/send_find_has_opened_email_to_candidates_worker.rb

```diff
@@ -1,11 +1,29 @@
 class SendFindHasOpenedEmailToCandidatesWorker
   include Sidekiq::Worker
 
+  STAGGER_OVER = 5.hours
+  BATCH_SIZE = 120
```

Comment:
any particular reason to have a slightly bigger batch size here?

& do these need to be constants?